### PR TITLE
custom-response: update configuration

### PR DESCRIPTION
**Pull request by TheConcierge**
Updating the new configuration method for `custom-response`

image

### CODE REVIEW

**Comment by ForgoneReality** [comment]
Looks like json under the yaml tab?

**Comment by TheConcierge** [comment]
> Looks like json under the yaml tab?

yes! so, in this specific example, we are sending a blob of type 'application/json' back. the content type is a string of json, but overall the format is still (valid) yaml!

**Review by ForgoneReality — APPROVED** [review]
in that case, LGTM!

File: docs/http/traffic-policy/actions/custom-response.mdx [hunk 1]
```diff
@@ -24,8 +24,9 @@ If this action is executed, no further actions in the traffic policy will be exe
...
-| `content`      | string | The body of the response.                                                                                          |
+| Parameter      | &nbsp;                    | Description                                                                                                                                                          |
+| -------------- | ------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `status_code`  | int                       | The status code of the response.                                                                                                                   |
+| `content`      | string                    | The body of the response.                                                                                                                          |
+| `headers`      | Map&lt;string, string&gt; | Headers to be added to the response. If `content-type` is not included in `headers`, this action will attempt to infer the correct `content-type`. |
```
Comment (rkolavo) [inline docs/http/traffic-policy/actions/custom-response.mdx:31]: @nijikokun do we want to explicitly call out that this will infer content-type here?

**Review by rkolavo — APPROVED** [review]
looks good to me, but would like input from @nijikokun on one thing

File: docs/http/traffic-policy/actions/custom-response.mdx (hunk 1 above)
Comment (TheConcierge) [reply docs/http/traffic-policy/actions/custom-response.mdx:31]: i was thinking it'd be good to document somewhere. maybe this isn't the place. also maybe we don't need to document it at all, maybe its absence is enough to imply to the user that they don't need it unless they really care

**Review by tammybailey — APPROVED** [review]

File: docs/http/traffic-policy/actions/custom-response.mdx (hunk 1 above)
Comment (nijikokun) [reply docs/http/traffic-policy/actions/custom-response.mdx:31]: Yes, this is good to know for our end users

**Review by nijikokun — APPROVED** [review]

**Pull request merged**